src/app/results/results.component.ts

```diff
@@ -81,7 +81,7 @@ export class ResultsComponent implements OnInit {
   }
 
   imageClick() {
-    this.getPresentPage(0);
+    this.getPresentPage(1);
     this.resultDisplay = 'images';
```

Comment:
The pagination bar used two kinds of indexing earlier, one was zero based for pages less than 4 and the other was one based for pages > 4, recently we modified the function and its calls to have just one kind of indexing so that it is cleaner and easier to understand, hence we need to update the indexing here too . Hope this helps make it clear :)